gatsby-theme-landing-page/src/styles/variables.module.css

```diff
@@ -18,21 +18,22 @@
   --letter-spacing-caps: 0.03em;
   --font-weight-light: 300;
   --font-weight-normal: 400;
+  --font-weight-semi-bold: 600;
```

Comment:
We can probably change the `bold` value to `600` for the demo, but keep the number of weights defined in the theme to 2 (assuming we're getting rid of `light`)